Sources/zkp/secp256k1.swift

```diff
@@ -151,16 +151,14 @@ extension secp256k1 {
 
     /// Backing initialization that creates a random secp256k1 private key for signing
     @usableFromInline init(format: secp256k1.Format = .compressed) throws {
```

Comment:
We will want to update the function signature so a try catch statement isn't needed.

```suggestion
    @usableFromInline init(format: secp256k1.Format = .compressed) {
```